src/bootloader/startup.c

```diff
@@ -56,8 +58,24 @@ int main(void)
 #endif
     bootloader_jump();
 
+    const uint8_t* hww_data = NULL;
+    uint8_t hww_frame[USB_REPORT_SIZE] = {0};
+
     // If did not jump to firmware code, begin USB processing
     while (1) {
+        if (hid_hww_read(&hww_frame[0])) {
```

Comment:
I don't mind this, but I wondered why you use this syntax over just `hww_frame` (saw it consistently in your code and was curious)

Reply:
`hww_frame` is a `uint8_t[64]` whereas `hid_hww_read` takes a `uint8_t*` I think modern compilers distinguish between these two types. But I could be wrong.

Reply:
Is this resolved?

Reply:
I think you might be wrong, the two are equivalent and you can just pass `hww_frame`.

Reply:
I just think it is clearer that I don't intend to pass the whole array as a value. But I'm open to change if you think it is less clear and you prefer the implicit decaying

Reply:
No, as I said at the top, I don't mind it at all, I was just curious.